react-debugging.html

```diff
@@ -63,27 +63,30 @@
           padding: "10px"
         }
         // No need to change anything ABOVE this line ^
-
+        
         return (
+          <>
           <h1>Draw a Card</h1>
-          <button onClick={}>
+          <button onClick={drawCard}>
             Click to Draw a Card
-          </button>
-          <button onclick={shuffle}>
-            Return Cards and Shuffle Deck
-          </button>
-          <div style={handStyles}>
-            {hand.map((card, index) => {
-              return (
-                <div style={cardStyles} key={index}>
-                  {card}
-                </div>
-              )
-            })}
-          </div>
-        )
-      }
+            </button>
+            <button onclick={shuffle}>
+              Return Cards and Shuffle Deck
+              </button>
+              <div style={handStyles}>
+                {hand.map((card, index) => {
+                  return (
+                    <div style={cardStyles} 
+                    key={index}>
+                    {card}
+                      </div>
+                      )
+                    })}
+                    </div>
+                    </>
 
+                    )
+                  }
```

Comment:
You missed one.  :(.  Load this in the browser and see what happens - then check the console for errors. :)